app/vulncode_app/models.py

```diff
@@ -50,13 +50,16 @@ class VulnerabilityReference(models.Model):
     class Meta:
         unique_together = ('vulnerability', 'source', 'reference_id')
 
+    def __str__(self):
+        return self.source
+
 
 class ImpactedPackage(models.Model):
     """
     Relates a vulnerability to package(s) impacted by it.
     """
     vulnerability = models.ForeignKey('Vulnerability')
-    package = models.ForeignKey('Package')
+    package_fk = models.ForeignKey('Package')
```

Comment:
You also likely need a unique together on these two fields to avoid dupe entries